### ISSUE TITLE
Add support for media queries

### ISSUE THREAD

**Issue by lancetharper** [Connormiha/jest-css-modules-transform#16]
Classes defined within `@media` are skipped. Some classes are only defined within media queries and they are omitted entirely.

Example with responsive utility classes:
```
.display-block {
    display: block;
}

@media (max-width: 600px) {
   .display-block\@sm {
        display: block;
   }
}
```

`display-block` is present in the result, but `display-block@sm` is not.

**Comment by Connormiha** [comment]
@lancetharper tnx for your request.
I fixed it in `v3.0.0`. Check it up.

**Comment by lancetharper** [comment]
Media queries work now! Thank you!

The update did break something that was working before:

```
.foo.bar {...}
```

Before this was split into `foo`, and `bar` but now the result is `foo.bar` and they aren't treated as separate classes.

**Comment by Connormiha** [comment]
fixed it in `v3.0.1`

**Comment by lancetharper** [comment]
Confirmed working. Thanks for the quick support! ⭐️
